tools/wasp-cli/chain/activate.go

```diff
@@ -56,7 +56,7 @@ func activateChain(ctx context.Context, node string, chainName string, chainID i
 
 		log.Check(err2)
 	} else {
-		_, err = client.ChainsAPI.ActivateChain(ctx, chainID.String()).Execute() //nolint:bodyclose // false positive
+		_, err = client.ChainsAPI.ActivateChain(ctx).Execute() //nolint:bodyclose // false positive
```

Comment:
Same here, keep the chainid.

Reply:
I will revert these change. Thank you for pointing out